src/custom-element.js

```diff
@@ -1604,8 +1604,8 @@ function createBaseCustomElementClass(win) {
       // 1. `noloading` attribute is specified;
       // 2. The element has not been whitelisted;
       // 3. The element is too small or has not yet been measured;
-      // 4. The element has already been laid out (include having loading
-      //    error);
+      // 4. The element has already been laid out, and does not support
+      //    reshowing the indicator (include having loading error);
```

Comment:
Optional: while we're here, can we reorder this comments to reflect the code?

Reply:
Sure thing!